test/e2e/upgrade_test.go

```diff
@@ -0,0 +1,61 @@
+// +build upgrade
+
+package e2e
+
+import (
+	"context"
+	"github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"os"
+	"strings"
+	"testing"
+)
+
+const EnvUpdateVersionKey = "UPDATE_TEST_VERSION"
+const upgradeTestTag = "next"
+
+func TestOperatorUpgrade(t *testing.T) {
+
+	upgradeTestVersion := os.Getenv(EnvUpdateVersionKey)
+	t.Log(upgradeTestVersion)
+
+	ctx, err := prepare(t)
+	if err != nil {
+		ctx.Cleanup()
+		require.FailNow(t, "Failed in prepare")
+	}
+	defer ctx.Cleanup()
+	addToFrameworkSchemeForSmokeTests(t)
+	if err := simplest(t, framework.Global, ctx); err != nil {
+		t.Fatal(err)
+	}
+	fw = framework.Global
+	createdJaeger := &v1.Jaeger{}
+	key := types.NamespacedName{Name: "my-jaeger", Namespace: ctx.GetID()}
```

Comment:
@rubenvp8510 I realize this is a short test, but I'd prefer if you followed the example of the other tests, used stretchr suites, and put all of this setup code in a SetupSuite().  That would make it consist with all of the other E2E tests